lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -738,7 +738,7 @@ export abstract class ClientApplication {
      * Helper to validate app environment before making an auth request
      * * @param interactionType
      */
-    protected preflightBrowserEnvironmentCheck(interactionType: InteractionType): void {
+    protected preflightBrowserEnvironmentCheck(interactionType: InteractionType, setInteractionInProgress: boolean = true): void {
```

Comment:
This is called from the 1p library for the broker scenarios, so there will be a separate PR in the 1p repo that will consume this change.